aptos-move/block-executor/src/scheduler.rs

```diff
@@ -81,13 +81,13 @@ pub enum ExecutionTaskType {
 
 /// A holder for potential task returned from the Scheduler. ExecutionTask and ValidationTask
 /// each contain a version of transaction that must be executed or validated, respectively.
-/// NoTask holds no task (similar None if we wrapped tasks in Option), and Done implies that
+/// Retry holds no task (similar None if we wrapped tasks in Option), and Done implies that
```

Comment:
Nit: these explanations can go together with the enum variants, then easier to keep comments up to date?
```
/// Holds no task (similar None if we wrapped tasks in Option).
Retry,
/// Implies that there are no more tasks and the scheduler is done.
Done,
```